# HG changeset patch
# Parent  de7718dbc13557b6c6a48689999b50abcfa6ccc0
Add build instructions for Debian like systems

diff --git a/CMakeLists.txt b/CMakeLists.txt
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -24,6 +24,7 @@
 # TODO: we need at least 2.0.19-stable because of the logging defines
 FIND_PACKAGE(Event2 REQUIRED COMPONENTS core)
 FIND_PACKAGE(Endian REQUIRED)
+FIND_PACKAGE(Sphinx)
 
 INCLUDE(CheckFunctionExists)
 INCLUDE(CheckVariableExists)
diff --git a/CMakeScripts/FindSphinx.cmake b/CMakeScripts/FindSphinx.cmake
new file mode 100644
--- /dev/null
+++ b/CMakeScripts/FindSphinx.cmake
@@ -0,0 +1,3 @@
+IF (NOT SPHINX)
+
+ENDIF ()
diff --git a/docs/first-steps.rst b/docs/first-steps.rst
--- a/docs/first-steps.rst
+++ b/docs/first-steps.rst
@@ -86,6 +86,31 @@
 Try to :ref:`toggle your lights <toggle>` and read on some of the examples
 bundled with lightsd.
 
+Linux (System V style)
+----------------------
+
+Start lightsd wiht:
+
+::
+
+   /etc/init.d/lightsd start
+
+Stop lightsd with:
+
+::
+
+   /etc/init.d/lightsd stop
+
+Check how lightsd is running with:
+
+::
+
+   ps aux | grep lightsd
+
+The logs will be logged to `syslogd(8)`_.
+
+.. _syslogd(8): http://manpages.debian.org/cgi-bin/man.cgi?query=syslogd&sektion=8
+
 Manually (other systems)
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
diff --git a/docs/installation.rst b/docs/installation.rst
--- a/docs/installation.rst
+++ b/docs/installation.rst
@@ -38,12 +38,12 @@
 Installation (for Arch Linux)
 -----------------------------
 
-Make sure you have yaourt installed: https://archlinux.fr/yaourt-en (`wiki
+Make sure you have Yaourt installed: https://archlinux.fr/yaourt-en (`wiki
 page`_).
 
 ::
 
-  yaourt -Sy lightsd
+  yaourt -Sya lightsd
 
 Make sure to follow the post-installation instructions: replace ``$USER`` with
 the user you usually use.
@@ -54,15 +54,66 @@
 
 ::
 
-  yaourt -Sy ipython
+  yaourt -Sya ipython
 
 Read on :doc:`/first-steps` to see how to use lightsd.
 
-.. _Yaourt: 
 .. _wiki page: https://wiki.archlinux.org/index.php/Yaourt
 
 .. _build_instructions:
 
+Build instructions for Debian/Ubuntu (dpkg based sytems)
+--------------------------------------------------------
+
+.. note:: Those instructions have been tested on Debian wheezy & jessie.
+
+Install the following requirements:
+
+::
+
+   build-essential cmake libevent-dev git ca-certificates ipython3 fakeroot wget devscripts debhelper
+
+Download and extract the release:
+
+.. code-block:: sh
+
+   wget -O lightsd_1.0.1.orig.tar.gz https://github.com/lopter/lightsd/archive/1.0.1.tar.gz
+   tar -xzf lightsd_1.0.1.orig.tar.gz
+   cd lightsd-1.0.1
+   wget -O - https://github.com/lopter/lightsd/releases/download/1.0.1/dpkg-1.0.1.tar.gz | tar -xzf -
+
+Build the package:
+
+::
+
+   debuild
+
+Install the package8
+
+.. note::
+
+   You will need to run this command as root with `sudo(8)`_ or be logged in as
+   root already.
+
+.. code-block:: sh
+
+   dpkg -i ../lightsd_1.0.1_`uname -r | cut -d- -f 3`.tar.gz
+
+Still as root, run the command the package asks you to run:
+
+.. note::
+
+   If you are not using sudo replace ``$USER`` by your regular non-root
+   username:
+
+.. code-block:: sh
+
+   gpasswd -a $USER lightsd
+
+Log out and back in as ``$USER`` for the change to take effect.
+
+.. _sudo(8): http://manpages.debian.org/cgi-bin/man.cgi?query=sudo&sektion=8
+
 Build instructions (for other systems)
 --------------------------------------
 
@@ -75,15 +126,8 @@
 lightsd is developed and tested from Arch Linux, Debian, OpenBSD and Mac OS X;
 both for 32/64 bits and little/big endian architectures.
 
-For Debian and Ubuntu you would need to install the following packages to build
-lightsd:
-
-::
-
-   build-essential cmake libevent-dev git ca-certificates
-
 Please also install ipython if you want to follow the examples in the next
-section. On Debian and Ubuntu you would install the ``ipython3`` package.
+section.
 
 From a terminal prompt, clone the repository and move to the root of it:
 
